packages/@tailwindcss-vite/src/index.ts

```diff
@@ -193,7 +192,7 @@ export default function tailwindcss(): Plugin[] {
           // The reason why we can not rely on the invalidation here is that the
           // users would otherwise see a flicker in the styles as the CSS might
           // be loaded with an invalid set of candidates first.
-          await server?.waitForRequestsIdle?.(id)
```

Comment:
We need to bump the minimum Vite version to the one where `waitForRequestsIdle` was introduced if it's not already btw.